client/src/pages/spectrogram/components/PluginsPane.tsx

```diff
@@ -158,7 +162,33 @@ export const PluginsPane = ({ cursorsEnabled, handleProcessTime, meta, setMeta }
           });
 
           setModalOpen(true);
+        } else {
+          // Data file
+          const samples_base64 = data.data_output[0]['samples'];
+          const samples = Buffer.from(samples_base64, 'base64');
+
+          const a = document.createElement('a');
+          document.body.appendChild(a);
```

Comment:
Do you need to append the element to the body ?

Reply:
I don't know - I copied this code from the siggen page. It works.